report/29_Process-Security.md

```diff
@@ -1 +1,45 @@
-Brief results of the security assessment.
+# Security
+## Risk Assessment
+
+![Risk assessment matrix](./images/risk_matrix.png)
+
+* Leaking user data
+  * **Description**: Most user data is public, but a few critical things such as emails and passwords are not. Leaking passwords are unlikely, but still technically possible for example due to logging misconfiguration, as they are stored only as hashes. The database is the most likely attack vector for accessing sensitive data.
```

Comment:
"Leaking passwords are unlikely, but still technically possible for example due to logging misconfiguration, as they are stored only as hashes."

Denne sætning er meget svær at læse. Burde det ikke være "Leaking passwords are unlikely as they are stored only as hashes, but it is still technically possible for example due to logging misconfiguration.